composer.json

```diff
@@ -112,7 +112,7 @@
         }
     },
     "require": {
-        "php": "^7.4 || ^8",
```

Comment:
Nope! We don't want to change this in the "main" branch.

Reply:
I have changed it for testing locally. I willll change it back.

Reply:
I have changed it back 😃